hazelcast/src/test/java/com/hazelcast/client/protocol/compatibility/ClientCompatibilityNullTest_2_0_1.java

```diff
@@ -54,14 +54,15 @@ public class ClientCompatibilityNullTest_2_0_1 {
     @Before
     public void setUp() throws IOException {
         File file = new File(getClass().getResource("/2.0.1.protocol.compatibility.null.binary").getFile());
-        InputStream inputStream = new FileInputStream(file);
-        byte[] data = new byte[(int) file.length()];
-        inputStream.read(data);
-        ByteBuffer buffer = ByteBuffer.wrap(data);
-        ClientMessageReader reader = new ClientMessageReader(0);
-        while (reader.readFrom(buffer, true)) {
-            clientMessages.add(reader.getClientMessage());
-            reader.reset();
+        try (InputStream inputStream = new FileInputStream(file)) {
+            byte[] data = new byte[(int) file.length()];
+            inputStream.read(data);
+            ByteBuffer buffer = ByteBuffer.wrap(data);
+            ClientMessageReader reader = new ClientMessageReader(0);
+            while (reader.readFrom(buffer, true)) {
+                clientMessages.add(reader.getClientMessage());
+                reader.reset();
```

Comment:
Should this not also be in try-with-resource?

Same comment applies to all `ClientCompatibilityNullTest`/`MemberCompatabilityTest`s

Reply:
ClientMessageReader  is not Java class, it is a HZ class. It does not have a close method

Reply:
> ClientMessageReader is not Java class, it is a HZ class. It does not have a close method

🤦